src/renderer/components/dock/log-resource-selector.tsx

```diff
@@ -30,36 +31,73 @@ export const LogResourceSelector = observer((props: Props) => {
     reload();
   };
 
-  const getSelectOptions = (containers: IPodContainer[]) => {
-    return containers.map(container => {
+  const onPodChange = (option: SelectOption) => {
+    const selectedPod = podsStore.getByName(option.value, pod.getNs());
+
+    if (!selectedPod) {
+      return;
+    }
+
+    const { getContainers, getInitContainers, getAllContainers } = selectedPod;
+
+    save({
+      selectedPod,
+      containers: getContainers(),
+      initContainers: getInitContainers(),
+      selectedContainer: getAllContainers()[0]
```

Comment:
Why store `containers` and `initContainers` at all? Since we can always get them on demand from `selectedPod`.

Reply:
Good call. I’ll make that change.